spec/paperclip/attachment_spec.rb

```diff
@@ -1477,6 +1478,53 @@ def call(filename)
     end
   end
 
+  context "an attachment with process_in_background set" do
+    before do
+      ActiveJob::Base.queue_adapter = :test
+      ActiveJob::Base.logger = nil
+
+      rebuild_model styles: { thumb: "100x100" },
+                    only_process: [:none],
```

Comment:
This is a hack to skip processing in real time. We need to think about better UI, so user can decide which styles he wants to be processed in real time/background